cds/modules/records/serializers/schemas/project.py

```diff
@@ -75,6 +74,7 @@ class ProjectSchema(StrictKeysSchema):
     title_translations = fields.Nested(TitleTranslationSchema, many=True)
     videos = fields.Nested(ProjectVideoSchema, many=True)
 
-    category = fields.Str()
-    type = fields.Str()
+    date = fields.Str()
+    category = fields.Str(required=True)
+    type = fields.Str(required=True)
```

Comment:
@drjova I don't know if the consistency is a good argument, as `type` probably means different things in different schema contexts. I'm not sure what is suppose to return for `Files`, but for the Project is `type=Movie|Rush|Video`). I agree `content_type` is probably more misleading, but maybe we can come up with a better name that is less generic than `type`. Meanwhile, if this is a blocker and you wish to move fast with this, we can also re-discuss it when we factor out the video part for Invenio.